### DISCUSSION TITLE
Verification email not being delivered.

### DISCUSSION THREAD

**Discussion by 3punt1415** [stoatchat/discussions#606] [General]
Hello,

I've recently been trying to make a Revolt account but my verification email is not being delivered.
I have resent the email around 10 times over several days.

**Comment by Rexogamer** [comment]
Heya! Have you opened a support ticket about this? If so, please give me the ticket ID - I can then check if your account has been manually verified/do so if not. If you haven't opened a ticket, please send an email to contact@revolt.chat (if you've already emailed us, you should already have a ticket open). Thanks!

(For further context, we recently received a ticket asking for manual verification, which we've done - if that was you, try logging in to your account again.)

**Reply by 3punt1415** [reply]
I had not yet, I'll do so right now.

**Reply by Rexogamer** [reply]
Turns out our messages to your mailserver are being blocked as spam. I've verified your account for you - thanks for getting in touch!
